src/createActionArgs.js

```diff
@@ -40,17 +40,37 @@ var createStateArg = function (action, model, isAsync, compute) {
   return state
 }
 
+var createServicesArg = function (action, services, parentKey) {
+  return Object.keys(services).reduce(function (newServices, key) {
+    if (typeof services[key] === 'function') {
+      newServices[key] = function () {
+        action.services.push({
+          name: parentKey || 'root',
+          method: key,
+          args: [].slice.call(arguments)
+        })
+        return services[key].apply(services[key], arguments)
+      }
+    } else {
+      newServices[key] = createServicesArg(action, services[key], key)
```

Comment:
current hoodie module wraps only store api. [compete hoodie-client api](https://github.com/hoodiehq/hoodie-client#api) would be a lot of painful and pointless wrapping.
I can wrap it with one method like `getHoodie(): { return hoodie }` but debugger will lost actual service run as well.
Can't we just leave non-function values untouched or wrapped with getter and setter?